src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -96,6 +99,25 @@
 
     Field2d field = new Field2d();
 
+    private final double[] yawMin = { 1, 2 };
+    private final double[] yawMax = { 1, 2 };
+    private final double[] pitchMin = { 1, 2 };
+    private final double[] pitchMax = { 1, 2 };
+
+    private enum Quadrant {
+        UNKNOWN("unknown"), REDA("redA"), REDB("redB"), BLUEA("blueA"), BLUEB("blueB");
```

Comment:
Do we not have better names than A and B for quadrant detection?

Reply:
The paths are called `Path A` and `Path B` in the manual. Hence `redA`, and `redB`. The text should probably be capitalized as `RedA` though.